config/webpack.settings.js

```diff
@@ -3,18 +3,19 @@ module.exports = {
 	entries: {
 		// JS files.
 		admin: './assets/js/admin/admin.js',
-		blocks: './assets/js/blocks/blocks.js',
 		frontend: './assets/js/frontend/frontend.js',
 		shared: './assets/js/shared/shared.js',
 		styleguide: './assets/js/styleguide/styleguide.js',
-		'blocks-editor': './includes/blocks/blocks-editor.js',
 
 		// CSS files.
 		'admin-style': './assets/css/admin/admin-style.css',
 		'editor-style': './assets/css/frontend/editor-style.css',
 		'shared-style': './assets/css/shared/shared-style.css',
 		style: './assets/css/frontend/style.css',
 		'styleguide-style': './assets/css/styleguide/styleguide.css',
+
+		// Blocks
+		'example-block': './includes/blocks/example-block/register.js',
```

Comment:
@ryanwelcher Would all custom blocks have it's own entry point?

Reply:
@samikeijonen yes. Technically, each block could have the same JS File as the enqueue process would only add it once but each block has it's own assets.